src/constants.ts

```diff
@@ -20,3 +20,9 @@ export const LIFECYCLE_STATES = {
   termination: 'terminate',
   unhold: 'unhold',
 }
+
+export const ISSUE_STATES = {
+  open: 'open',
+  closed: 'closed',
+  all: 'all',
+} as const;
```

Comment:
I would define this only as type and derive it directly from the GitHub API.
We only use it as non-type in the `GitHubDeploymentManager`, and for one spot I believe an ENUM-kind-of-definition is too much.

Given they `GitHubDeploymentManager` already has a dependency to `octokit.js`, lets derive the issue state from there and then just use a string as the default value (I will leave a comment on those code references as well.)